doc/developer-documentation/forgae-cli/compile.md

```diff
@@ -8,6 +8,18 @@ forgae compile
 
 The **compile** command compiles Sophia contract. It's recommended to use **.aes** file extension. Default directory is $projectDir/contracts. The result of the compilation is the contract bytecode printed in the console. Additional **--path** parameter is available, which can specify the path to the contract to be compiled.
 
+You can specify network using the **-n** or **--network** option. There are 3 options for networks predefined and available:
```

Comment:
as far as I remember we don't need this anymore? I think that we have removed it recently? @alekseytsekov  am I right?

Reply:
@maptuhec yes, you are right.